File: samples/Java/Json Store/Cursor/Sort Query Result.java
Store store = session.getDatabase("playground").getStore("owners");
String s = ""; final StringBuilder b = new StringBuilder();

s += ">>> Sort documents by first name, limited to 5\n";
Cursor c = store.openCursor().range(0,5).orderBy("@firstName");
b.setLength(0);
c.find(new CursorHandler() {
  public boolean handle(CursorEntry e) throws JsonException {  
  	b.append("  "+e.getString("firstName")+" "+e.getString("lastName")+"\n");
    return true;
  }
});  
s += b.toString()+"\n";

s += ">>> Sort documents by first name, descending, limited to 5\n";
c = store.openCursor().range(0,5).orderBy("@firstName desc");
b.setLength(0);
c.find(new CursorHandler() {
  public boolean handle(CursorEntry e) throws JsonException {  
  	b.append("  "+e.getString("firstName")+" "+e.getString("lastName")+"\n");
    return true;
  }
});  
s += b.toString()+"\n";


_formatText("{0}",s);

File: samples/JavaScript/Json Store/Cursor/Sort Query Result.js
session.setAsync(false);

var store = session.getDatabase("playground").getStore("owners");
var s = "";

s += ">>> Sort documents by first name, limited to 5\n";
var c = store.openCursor().range(0,5).orderBy('@firstName');
c.find(function(e) {
  s += "  "+e.getString("firstName")+" "+e.getString("lastName")+"\n";
});  
s += "\n";

s += ">>> Sort documents by first name, descending, limited to 5\n";
var c = store.openCursor().range(0,5).orderBy('@firstName desc');
c.find(function(e) {
  s += "  "+e.getString("firstName")+" "+e.getString("lastName")+"\n";
});  
s += "\n";


darwino.Utils.setText("content","{0}",s);